Rakefile

```diff
@@ -50,17 +50,17 @@ namespace :crowbar do
     ).flatten.compact.uniq
   end
 
-  desc "Init all barclamps"
+  desc "Initialize all barclamps (:fork, :clone, :add_upstream, :add_susecloud)"
```

Comment:
me too but others maybe find it easier to understand. I wanted to be more verbose but not too verbose.